tests/phpunit/tests/ajax/CropImage.php

```diff
@@ -0,0 +1,158 @@
+		}
+
+		$response = json_decode( $this->_last_response, true );
+		$this->validate_response( $response );
+
+		$cropped_attachment = get_post( $response['data']['id'] );
+		$this->assertInstanceOf( WP_Post::class, $cropped_attachment, 'get_post function must return an instance of WP_Post class' );
+		$this->assertNotEmpty( $attachment->post_title, 'post_title value must not be empty for testing purposes' );
+		$this->assertNotEmpty( $cropped_attachment->post_title, 'post_title value must not be empty for testing purposes' );
+		$this->assertSame( $attachment->post_title, $cropped_attachment->post_title, 'post_title value should be copied over to the cropped attachment' );
+		$this->assertSame( $attachment->post_content, $cropped_attachment->post_content, 'post_content value should be copied over to the cropped attachment' );
+		$this->assertSame( $attachment->post_excerpt, $cropped_attachment->post_excerpt, 'post_excerpt value should be copied over to the cropped attachment' );
+		$this->assertSame( $attachment->_wp_attachment_image_alt, $cropped_attachment->_wp_attachment_image_alt, '_wp_attachment_image_alt value should be copied over to the cropped attachment' );
+
+		wp_delete_attachment( $attachment->ID, true );
+		wp_delete_attachment( $cropped_attachment->ID, true );
+	}
+
+	/**
+	 * Tests that attachment properties are not auto-generated if they are not defined for the original image.
+	 *
+	 * @ticket 37750
+	 *
+	 * @covers ::wp_ajax_crop_image
+	 */
+	public function test_it_doesnt_generate_new_metadata_if_metadata_is_empty() {
+		// Become an administrator.
+		$this->_setRole( 'administrator' );
+
+		$attachment = $this->create_attachment( false );
+		$this->prepare_post( $attachment );
+
+		// Make the request.
+		try {
+			$this->_handleAjax( 'crop-image' );
+		} catch ( WPAjaxDieContinueException $e ) {
+		}
+
+		$response = json_decode( $this->_last_response, true );
+		$this->validate_response( $response );
+
+		$cropped_attachment = get_post( $response['data']['id'] );
+		$this->assertInstanceOf( WP_Post::class, $cropped_attachment, 'get_post function must return an instance of WP_Post class' );
+		$this->assertEmpty( $attachment->post_title, 'post_title value must be empty for testing purposes' );
+		$this->assertNotEmpty( $cropped_attachment->post_title, 'post_title value must be auto-generated if it\'s empty in the original attachment' );
+		$this->assertStringStartsWith( 'http', $cropped_attachment->post_content, 'post_content value should contain an URL if it\'s empty in the original attachment' );
+		$this->assertEmpty( $cropped_attachment->post_excerpt, 'post_excerpt value must be empty if it\'s empty in the original attachment' );
+		$this->assertEmpty( $cropped_attachment->_wp_attachment_image_alt, '_wp_attachment_image_alt value must be empty if it\'s empty in the original attachment' );
+
+		wp_delete_attachment( $attachment->ID, true );
+		wp_delete_attachment( $cropped_attachment->ID, true );
```

Comment:
same note as previous test.

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/commit/c55674c7b8e71c6043f2bd6aa35cf42d5c6d51be